src/main/resources/swagger/api-docs.yaml

```diff
@@ -10,6 +10,11 @@ info:
     url: http://opensource.org/licenses/BSD-3-Clause
   termsOfService: https://github.com/broadinstitute/firecloud-orchestration
 
+host: api.firecloud.org
```

Comment:
this change causes some issues when using this on non-production environments as all calls from the swagger on non-prod environments will be sent to production orchestration.

we would likely need to template this in somehow in order to get the desired behavior. i.e., we have a mapping of environments to hosts:

local -> local.broadinstitute.org:10443
dev -> firecloud-orchestration.dsde-dev.broadinstitute.org
alpha -> firecloud-orchestration.dsde-alpha.broadinstitute.org
firecloud in a box (dev) -> firecloud-fiab.dsde-dev.broadinstitute.org:23443
firecloud in a box (qa) -> firecloud-fiab.dsde-qa.broadinstitute.org:23443
...and so on...
prod -> api.firecloud.org

Reply:
OK, removed host and schemes again.